{{ cookiecutter.project_slug }}/flows/index.yml

```diff
@@ -3,20 +3,22 @@ with:
   logserver: true
 pods:
   crafter:
-    uses_internal: pods/craft.yml
+    uses: pods/craft.yml
+    parallel: $PARALLEL
     read_only: true
   encoder:
-    uses_internal: pods/encode.yml
+    uses: pods/encode.yml
     parallel: $PARALLEL
     timeout_ready: 600000
     read_only: true
   chunk_idx:
-    uses_internal: pods/chunk.yml
+    uses: pods/chunk.yml
     shards: $SHARDS
     separated_workspace: true
   doc_idx:
-    uses_internal: pods/doc.yml
-    needs: gateway
+    uses: pods/doc.yml
```

Comment:
From the migration guide: yaml_path and image are unified into one single keyword: ```uses```. When a Pod running inside a docker container used an inner yaml_path, it will now use ```uses_internal```.
Depending on whether docker container is used, this could be modified accordingly.
We could have comment mentioning this since the preference is to use ```uses```